requirements.txt

```diff
@@ -6,7 +6,7 @@ opencv-python==4.5.1.48 ; platform_machine != "aarch64" and platform_machine !=
 opencv-contrib-python==4.5.4.58 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and python_version == "3.10"
 opencv-contrib-python==4.5.1.48 ; platform_machine != "aarch64" and platform_machine != "armv6l" and platform_machine != "armv7l" and python_version != "3.10"
 opencv-python==4.4.0.46 ; platform_machine == "armv6l" or platform_machine == "armv7l"
-opencv-contrib-python==4.4.0.46 ; platform_machine == "armv6l" or platform_machine == "armv7l"
+opencv-contrib-python==4.1.0.25 ; platform_machine == "armv6l" or platform_machine == "armv7l"
```

Comment:
Why? Test code? 
Discrepancy between opencv python and contrib python might cause issues.